bitsandbytes/cuda_setup/env_vars.py

```diff
@@ -26,7 +26,7 @@ def to_be_ignored(env_var: str, value: str) -> bool:
 
 
 def might_contain_a_path(candidate: str) -> bool:
-    return "/" in candidate
+    return os.sep in candidate
```

Comment:
@Titus-von-Koeller you worked with universal paths before. Do you see a problem with this? I think once we separate out the `:` (or other divisors) we could use the PosixPath instead of this.

Reply:
this looks good to me